padmy/run.py

```
@@ -169,6 +169,44 @@ async def compare_db_main(
     pprint_compared_dbs(db1, db2)
 
 
+@cli.command(cmd="dump", help="Dump a database using pg_dump")
+def dump_main(
+    pg_infos: PGConnectionInfo = Derived(get_pg_infos),
```

Ruff (0.14.7)

Do not perform function call `Derived` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)